flink-core/src/main/java/org/apache/flink/api/common/cache/DistributedCache.java

```diff
@@ -15,8 +15,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package org.apache.flink.api.common.cache;
 
+import org.apache.flink.annotation.Public;
+import org.apache.flink.configuration.Configuration;
```

Comment:
If you make this file checkstyle-compliant please also remove the exclusion in `tools/maven/suppressions-core.xml`

Reply:
Sure, I wondered why it did not fail the build before...